packages/graphback-resolvers/src/ResolversCRUDPlugin.ts

```diff
@@ -0,0 +1,199 @@
+import { GraphbackPlugin, GraphbackCoreMetadata, getFieldName, getSubscriptionName, GraphbackOperationType, ModelDefinition, getPrimaryKey, FieldRelationshipMetadata, getDeltaQuery } from '@graphback/core';
+import { IResolvers } from '@graphql-tools/utils'
+
+
+/**
+ * Generate runtime resolver layer using Apollo GraphQL format
+ * and injected service layer. Service layer offers various capabilities like monitoring, cache etc.
+ * so resolver logic can be kept simple and interchangable.
+ *
+ * Resolvers are formatted using graphql-tools format
+ *
+ * ```javascript
+ * const resolvers = {
+ *    Query: {...}
+ *    Mutation: {...}
+ *    Subscription: {...}
+ * }
+ * ```
+ */
+export class ResolversCRUDPlugin extends GraphbackPlugin {
+
+  public addResolvers(metadata: GraphbackCoreMetadata): IResolvers {
+    const resolvers: IResolvers = {
```

Comment:
This is simply a port of [LayeredRuntimeResolverCreator](https://github.com/aerogear/graphback/blob/61b24b7b9b83bf361bdacb754d5a32efeea4833f/packages/graphback-runtime/tests/LayeredRuntimeResolverCreatorTest.ts#L25)